File: API/Models/Domain/Answer.cs
using API.Models.Domain.Questions;
using Newtonsoft.Json;

namespace API.Models.Domain
{

   
    public class Answer
    {
        

        public int Id { get; set; }

        public string AnswerText { get; set; }

        public bool IsCorrect { get; set; }





        public int QuestionId { get; set; }



        public virtual Question Question { get; set; }

        public string UserId { get; set; }
        
    }
}

File: client2/src/app/types/answer.ts
enum AnswerType {
  Choice,
  MultipleChoice,
  TrueFalse,
  Basic
}


export type Answer= {
    Id: number,
    AnswerText: string,
    IsCorrect: boolean,
    Type: AnswerType ,
    QuestionId: number,
    UserId: string,
}
